resolver/oneInchmulti.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+export async function ExecuteQuote(quoteParams) {
+  for (var i = 0; i < quoteParams.fromTokenAddressArray.length; i++) {
+    var singlequoteParams = {
+      fromTokenAddress: quoteParams.fromTokenAddressArray[i],
+      toTokenAddress: quoteParams.toTokenAddressArray[i],
+      amount: quoteParams.amountArray[i],
+    };
+    const chainId = 56; //For Binance smart chain
```

Comment:
It should be an object and based on the network, and we should select the correct chainID.

Reply:
Done